### DISCUSSION TITLE
Is there a release policy?

### DISCUSSION THREAD

**Discussion by el-hult** [plotters-rs/plotters#568] [Q&A] [answered]
Latest release of plotters (0.3.5) was 10 months ago. But there are nice updates on the `master` branch that are really useful to me. Are you planning on a 0.3.6 anytime soon? Are you releasing from `master` or from `next-release-devel`?

To make the question slightly broader: do you have any general plan for the relase process going forward?

Best,
Ludvig

**Comment by AaronErhardt** [comment, marked as answer]
There is no policy and the project is severely understaffed (as often in FOSS): https://github.com/plotters-rs/plotters/issues/345

I can make a release soon though.

**Reply by el-hult** [reply]
Ok cool. Don't stress it. The code is available here on github, so I'm fine :). You hosting it here is a huge service! Thank you!

If I start to use the project more intensely, I'll see if I can help you out!
